packages/html-reporter/src/hideIfErrors.tsx

```diff
@@ -13,19 +13,20 @@
  * See the License for the specific language governing permissions and
```

Comment:
This diff is odd. GitHub thinks the file moved. But really: `globalSetup.ts` was deleted, and `hideIfErrors.tsx` was added. They are not related at all, and hideIfErrors.tsx is net new, so just look at the file, not the diff.